### DISCUSSION TITLE
"request entity too large" with ~150kB of text

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#4539] [Archive]
(This one is about v9.0.0-rc.50 with Docker)

When adding about 150 kB of text to an item via the web ui and trying to save it, the error `request entity too large` appears and the data isn't saved.

In other cases I would change the setting in express or else, but where to change the setting here?

Thanks a lot,
Robert

**Comment by rijkvanzanten** [comment]
Heya! This can be configured with the [`MAX_PAYLOAD_SIZE`](https://docs.directus.io/reference/environment-variables/#general)  environment variable 👍🏻

**Comment by sagerio** [comment]
Shame on me... Why I didn't found this in the docs... 
Thanks so much!
